x-pack/test/functional/apps/ml/data_frame_analytics/classification_creation.ts

```diff
@@ -167,10 +168,37 @@ export default function ({ getService }: FtrProviderContext) {
           );
         });
 
+        it('should open the edit form for the created job in the analytics table', async () => {
+          await ml.dataFrameAnalyticsTable.filterWithSearchString(testData.jobId);
+          const rows = await ml.dataFrameAnalyticsTable.parseAnalyticsTable();
+          const filteredRows = rows.filter((row) => row.id === testData.jobId);
+          expect(filteredRows).to.have.length(
+            1,
+            `Filtered analytics table should have 1 row for job id '${testData.jobId}' (got matching items '${filteredRows}')`
+          );
+          if (filteredRows?.length) {
```

Comment:
Not sure why we need this check here. If the job list filtering didn't succeed, the test failed earlier anyway.
(same for outlier detection and regression)

Reply:
Removed in 64abf306dfb6f2021e1536a2542b4631ba07de6a